database/migrations/2017_10_14_083556_change_gift_column_structure.php

```diff
@@ -15,7 +15,12 @@ class ChangeGiftColumnStructure extends Migration
     public function up()
     {
         Schema::table('gifts', function (Blueprint $table) {
+          if (get_class(DB::connection()) == "Illuminate\Database\PostgresConnection") {
```

Comment:
You can do `if (DB::connection()->getDriverName() == 'pgsql')`

Reply:
Ah thanks! I was actually looking for this but was unable to find something like it. I will update the PR!